src/components/SvgLibraries/shared/NoResult.js

```diff
@@ -44,14 +45,16 @@ const NoResult = ({
         </p>
         <Row className="resource-card-group">
           <Column colMd={4} colLg={4} noGutterSm>
-            <ResourceCard subTitle="Submit an icon design">
+            <ResourceCard subTitle={`Submit a ${pageName} design`}
```

Comment:
should be `an icon` not `a icon`
```suggestion
            <ResourceCard subTitle={`Submit ${pageName === 'icon' ? 'an icon' : 'a pictogram'} design`}
```